src/test/java/calculator/domain/PositiveNumber.java

```diff
@@ -0,0 +1,47 @@
+package calculator.domain;
+
+import calculator.common.exception.InvalidArgumentException;
+import calculator.common.exception.InvalidArgumentExceptionMessage;
+
+public class PositiveNumber {
+
+    private static final int MINIMUM_NUMBER = 0;
+    private static final PositiveNumber ZERO = new PositiveNumber(MINIMUM_NUMBER);
+    private final int number;
+
+    public PositiveNumber(final int number) {
+        this.number = number;
+        validation();
+    }
+
+    private void validation() {
+        isPositive();
+    }
+
+    private void isPositive() {
+        if (number < 0) {
```

Comment:
```suggestion
        if (number < MINIMUM_NUMBER) {
```
상수로 만든 만큼 활용해 보는 것이 좋을 것 같아요~